com.unity.render-pipelines.core/Documentation~/Common/light-anchor.md

```diff
@@ -0,0 +1,16 @@
+# Light Anchor
+
+![](Images/LightAnchor0.png)
+
+A Light Anchor must be conncted to a Light. Which allow us to light a scene on a Rendered Camera Space.
+
+## Properties
+
+| **Property**    | **Description**                                              |
+| --------------- | ------------------------------------------------------------ |
+| Orbit | Control the Orbit of the light, can be controled with the first green knob. |
+| Evevation | Control the Elevation of the light, can be controled with the second blue knob. |
+| Roll | Control the roll, usefull if the light had an IES or a Cookie, can be controled with the third gray knob. |
+| Distance | Controls how far 'back', the light is placed from its anchor. |
```

Comment:
| Distance | Controls the distance between the light and its anchor in world space. |